sdk/nodejs/src/environmentConfigManager.ts

```diff
@@ -103,9 +103,10 @@ export class EnvironmentConfigManager {
             return
         } else if (res?.status === 200 && projectConfig) {
             try {
+                const etag = res?.headers.get('etag') || ''
                 getBucketingLib().setConfigData(this.sdkKey, projectConfig)
```

Comment:
were you intending to pass it in here?

Reply:
I was during testing, but need to wait to release the WASM code first b4 I can update the NodeJS SDK as its WASM version is pinned.